### ISSUE TITLE
WCRFT or Toki integration

### ISSUE THREAD

**Issue by djstrong** [CLARIN-PL/PolDeepNer#1]
There is no integration with WCRFT or Toki in docker image, am I right?

**Comment by mczuk** [comment]
Yes, you are right. At the moment there is only a naive tokenization. Hopefully, it will be added soon :-)
